dind/ros2/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG ROS2_DISTRO=foxy
 
-FROM ros:${ROS2_DISTRO}
+FROM wamvtan/ros2-ci-base:${ROS2_DISTRO}
```

Comment:
can you specify base image "wamvtan/ros2-ci-base" as argument?

Reply:
I added a new argument “IMAGE_NAME” for it just now.